lib/tasks/import_product_images.rake

```diff
@@ -1,11 +1,11 @@
 # frozen_string_literal: true
 
+COLUMNS = %i[producer name image_url].freeze
+
 namespace :ofn do
   namespace :import do
     desc "Importing images for products from CSV"
     task :product_images, [:filename] => [:environment] do |_task, args|
-      COLUMNS = [:producer, :name, :image_url].freeze
-
```

Comment:
Is this now a global constant? I wonder if the real mistake was the naming so that it becomes a constant. Can we use a simple local variable here instead?